tomcat/src/main/java/camp/nextstep/service/UserService.java

```diff
@@ -8,7 +8,11 @@
 
 public class UserService {
 
-    public User findLoginUser(final Session session) {
+    public boolean isNotLogin(final Session session) {
+        return findLoginUser(session) != null;
+    }
+
+    private User findLoginUser(final Session session) {
```

Comment:
private 함수까지 Session정보가 그대로 전달되는 것 보다는 해당 메서드에서 필요로하는 인자를 뽑아서 전달드리는 방식을 권장드려요. 켄트백의 Tidy First라는 책에서는 명시적인 매개변수라는 챕터로 이 내용을 설명하는데요. 함수에서 필요한 매개변수를 블록(ex: Map)으로 매개변수가 전달되는 경우 해당 함수가 실제로 어떤 데이터가 필요한지를 숨기기 때문에 가독성이 떨어지는 이슈가 있거든요. 

```suggestion
    public boolean isNotLogin(final Session session) {
        final Object userAttr = session.getAttribute("user");
        return findLoginUser(userAttr) != null;
    }

    private User findLoginUser(final Object userAttr) {
```